project/Dependencies.scala

```diff
@@ -106,4 +106,7 @@ object Dependencies {
     workbenchMetrics,
     sam
   )
+
+  val modelDependencies = rootDependencies
```

Comment:
`model` probably doesn't need all these dependencies.

Reply:
almost certainly. i was trying to avoid the extremely boring job and Just Make It Work